### PR TITLE
Header component

### PR DESCRIPTION
**Pull request by MilosIl**
Needs to change stuff when we will have auth configured. I used latest changes(singin), since dev branch wasn't up to date

### CODE REVIEW

**Review by Ben2oid — APPROVED** [review]
Looks good to me. As a first iteration, it does it's job. We'll need to make the header to react to scrolling too down the lane later down the road.

File: src/pages/Login/Login.tsx [hunk 1]
```diff
@@ -12,25 +13,62 @@ export const Login = () => {
...
+            </p>
+            <div className='createAcc'>
+              <p>Create an account</p>
+              <Link style={{ textAlign: 'right' }} to='/signup'>
+                sign up
+              </Link>
+            </div>
+          </div>
+          <div className='rightSide'>
+            <img src='public\images\girlInCar.png' alt='' />
```
Comment (nikaakin) [inline src/pages/Login/Login.tsx:68]: give this image some alt

File: src/pages/Login/Login.tsx [hunk 2]
```diff
@@ -12,25 +13,62 @@ export const Login = () => {
...
+            </form>
+            <p style={{ textAlign: 'center' }}>or</p>
+            <p>
+              This site is protected by reCAPTCHA and the Google
+              <a href='placeholder'>Privacy Policy</a>
+              and<a href='placeholder'>Terms of Service</a>apply.
+            </p>
+            <div className='createAcc'>
+              <p>Create an account</p>
+              <Link style={{ textAlign: 'right' }} to='/signup'>
```
Comment (nikaakin) [inline src/pages/Login/Login.tsx:62]: lets try not to use inline style if we can use classes

File: src/pages/Login/Login.tsx [hunk 3]
```diff
@@ -12,25 +13,62 @@ export const Login = () => {
...
+                  })}
+                  placeholder={'Password'}
+                  type='password'
+                />
+              </div>
+
+              <p>{errors.password?.message}</p>
+              <Link to=''>Forgot Your Password?</Link>
+              <p></p>
+              <Button type='submit' children={'Sign in'} className={'btn'} />
```
Comment (nikaakin) [inline src/pages/Login/Login.tsx:52]: So this is minor stuff but when componen has property children is mostly means you are passing another component as child component. If you dont mind lets stick to this. If you wanted just a display text for button class that property title, text or content. Also {'btn'}  for properties its not neccessary to give additional "{}" to just for string value. Again this is minor but I think it look better that way, so you decide what to do with those

File: src/pages/Login/Login.tsx [hunk 4]
```diff
@@ -12,25 +13,62 @@ export const Login = () => {
...
-      <p>
-        Not yet registered? <Link to='/signup'>Click here</Link>
-      </p>
-    </form>
+    <>
+      <Header />
+      <div className='container'>
+        <div className='midContainer'>
+          <div className='leftSide'>
+            <h1 style={{ fontWeight: '700', fontSize: '48px' }}>Sign In</h1>
```
Comment (nikaakin) [inline src/pages/Login/Login.tsx:21]: lets try not to use inline style if we can use classes

File: src/pages/Login/Login.tsx [hunk 5]
```diff
@@ -12,25 +13,62 @@ export const Login = () => {
...
+                />
+                <input
+                  {...register('email', { required: 'Email is required' })}
+                  placeholder={'Email Address'}
+                />
+              </div>
+
+              <p>{errors.email?.message}</p>
+              <div className='inputDiv'>
+                <img className='inputIcon' src='public\images\eye.svg' alt='' />
```
Comment (nikaakin) [inline src/pages/Login/Login.tsx:37]: fill alt tag

File: src/pages/Login/Login.tsx [hunk 6]
```diff
@@ -12,25 +13,62 @@ export const Login = () => {
...
+                <input
+                  className='inputPassword'
+                  {...register('password', {
+                    required: 'Password is required',
```
Comment (nikaakin) [inline src/pages/Login/Login.tsx:41]: for now this is fine but in future we won't be typing form validation here. we will make schema file and keep them in one place. I'll initialize those as well when you tell me

**Comment by nikaakin** [comment]
I mainly added few stuff that would probably make code look better so if you feel like you dont need to change those just ping me again from reviews tab and I'll approve

**Review by nikaakin — APPROVED** [review]

**Pull request merged**